resources/views/components/po-header.blade.php

```diff
@@ -67,7 +67,7 @@
         </div>
         <a href="{{ route('po-pdf', $purchaseOrder) }}" class="btn btn-primary m-1 w-100 d-block">Download
             PDF</a>
-        <a href="{{ route('receiving_orders.create') }}" class="btn btn-warning m-1 w-100 d-block">Receive</a>
+        <a href="#" class="btn btn-warning m-1 w-100 d-block disabled">Receive</a>
```

Comment:
Receiving order functionality not implemented. Disabled link and removet non-existing route.